backend/api/migrations/0002_auto_20170310_2142.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.1 on 2017-03-10 21:42
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import csv
+
+def load_initial_data(apps, schema_editor):
+    Business = apps.get_model("api", "Business")
+    Inspection = apps.get_model("api", "Inspection")
+
+    with open('../../../DOHMH_NYC_Restaurant_Inspection_Results.csv') as csv_file:
+        reader = csv.reader(csv_file, delimiter=',')
+        header = next(reader)
+        for row in reader:
```

Comment:
Very smart idea to do a `bulk_create` here but I don't think you're using it correctly. You're calling `bulk_create` inside the loop and passing it a list of length 1 each time, which is basically the same thing as calling `save` 65k times except more error-prone.

Instead, you should use the loop to build up a list of `Business` objects (checking `camis` id each time so you don't add repeats) and separately a list of `Inspection` objects. Don't need to worry about repeats for `Inspections` obviously since they are already guaranteed to be unique.

Then once the loop has finished, call `bulk_create` once for your businesses and then a second time for your inspections. Make sense?